styles.css

```diff
@@ -0,0 +1,80 @@
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  display: flex;
+  justify-content: space-between;
+  margin: 24px;
+}
+
+.logo a {
+  text-decoration: none;
+}
+
+.logo h3 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 20px;
+  color: #6070ff;
+  line-height: 29px;
+}
+
+.Hamburger {
+  border: none;
+  background-color: white;
+  border-color: white;
+  cursor: pointer;
+  width: px;
+}
+
+.Hamburger img {
+  width: 18px;
+}
+
+.header-Page {
+  background-size: 45%;
+  margin: 84px 24px 24px 24px;
+}
+
+.header-Bg {
+  background-image: url(images/header-bg.png);
+  background-position: top left;
+  background-repeat: no-repeat;
+  background-size: auto;
+  overflow: hidden;
+}
+
+.header-Page h1 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  margin: 24px 0 24px 0;
+}
+
+.header-Page p {
+  margin: 24px 0 24px 0;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.header-Page h6 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.socialMedia {
+  width: 100%;
+  margin: 5px 0 0 -5px;
+}
```

Comment:
- [ ] As a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) you should use Flexbox to align your content. This gives us a lot of options to align all your elements and it's responsive. To learn more you can check this [page ](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) :+1: